### PR TITLE
Add: setting a cron job section in the README

### PR DESCRIPTION
**Pull request by david-guyon**
Pull-request in answer to the issue #20.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,3 +225,25 @@ nova keypair-list
 (laptop) ssh -NL 8000:parapluie-32-kavlan-16.rennes.grid5000.fr:80 access.grid5000.fr
 ```
 And then visit ```http://127.0.0.1:8000/horizon```
+
+## Miscellaneous
```
Comment (msimonin) [inline README.md:231]: Could you add a direct link in the menu section : in the beginning of the README ?

File: README.md [hunk 2]
```diff
@@ -225,3 +225,25 @@ nova keypair-list
...
+
+### Setting a cron job
```
Comment (msimonin) [inline README.md:233]: Maybe here as well :)

**Comment by david-guyon** [comment]
Here you go ;). 

Hum about `git rebase`, I'm kind of confused of what value I should put at the beginning of these lines:

```
1 pick f65779b Add: setting a cron job section in the README
2 pick 4df129f Add: menu entries for Miscellaneous section
```

**Comment by msimonin** [comment]
```
1 pick f65779b Add: setting a cron job section in the README
2 f 4df129f Add: menu entries for Miscellaneous section
```

**Comment by david-guyon** [comment]
Improving my Git skills every day since I started the internship ;). 

If you agree with the content of this pull-request, I think it's done now! :)

**Pull request merged**